### PR TITLE
android, desktop: notes to self

### PR DESCRIPTION
**Pull request by avently**

### CODE REVIEW

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/ui/theme/Color.kt [hunk 1]
```diff
@@ -29,3 +30,4 @@ val FileLight = Color(183, 190, 199, 255)
 val FileDark = Color(101, 101, 106, 255)
 
 val MenuTextColor: Color @Composable get () = if (isInDarkTheme()) LocalContentColor.current.copy(alpha = 0.8f) else Color.Black
+val NoteFolderIconColor: Color @Composable get() = MaterialTheme.colors.primary.copy(0.33f)
```
Comment (epoberezkin) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/ui/theme/Color.kt:33]: eol

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/item/CIFileView.kt [hunk 2]
```diff
@@ -81,7 +81,7 @@ fun CIFileView(
         }
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/item/CIFileView.kt:81]: doesn't open save dialogue on tap (not via menu but on chat item tap)

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatPreviewView.kt [hunk 3]
```diff
@@ -305,7 +305,7 @@ fun ChatPreviewView(
             color = Color.White,
             fontSize = 11.sp,
             modifier = Modifier
-              .background(if (disabled || showNtfsIcon) MaterialTheme.colors.secondary else MaterialTheme.colors.primaryVariant, shape = CircleShape)
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatPreviewView.kt:308]: why secondary if marked unread? same on ios

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt [hunk 4]
```diff
@@ -681,6 +681,19 @@ object ChatController {
       }
     }
   }
+   suspend fun apiCreateChatItem(rh: Long?, noteFolderId: Long, file: CryptoFile? = null, mc: MsgContent): AChatItem? {
+    val cmd = CC.ApiCreateChatItem(noteFolderId, file, mc)
+    val r = sendCmd(rh, cmd)
+    return when (r) {
+      is CR.NewChatItem -> r.chatItem
+      else -> {
+        if (!(networkErrorAlert(r))) {
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt:690]: unnecessary to check for network error I think, there should be no network IO

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatItemInfoView.kt [hunk 5]
```diff
@@ -156,7 +156,7 @@ fun ChatItemInfoView(chatModel: ChatModel, ci: ChatItem, ciInfo: ChatItemInfo, d
   fun Details() {
     AppBarTitle(stringResource(if (sent) MR.strings.sent_message else MR.strings.received_message))
     SectionView {
-      InfoRow(stringResource(MR.strings.info_row_sent_at), localTimestamp(ci.meta.itemTs))
+      InfoRow(stringResource(if (!ci.localNote) MR.strings.info_row_sent_at else MR.strings.info_row_created_at), localTimestamp(ci.meta.itemTs))
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatItemInfoView.kt:159]: in share text it says "sent at"

Comment (spaced4ndy) [reply apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatItemInfoView.kt:159]: and perhaps title in view and share text should say "Saved message", not "Sent message"?

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/item/CIFileView.kt [hunk 6]
```diff
@@ -152,10 +152,11 @@ fun CIFileView(
             when (file.fileProtocol) {
               FileProtocol.XFTP -> progressIndicator()
               FileProtocol.SMP -> fileIcon()
+              FileProtocol.LOCAL -> fileIcon()
             }
           is CIFileStatus.SndTransfer ->
             when (file.fileProtocol) {
-              FileProtocol.XFTP -> progressCircle(file.fileStatus.sndProgress, file.fileStatus.sndTotal)
+              FileProtocol.XFTP, FileProtocol.LOCAL -> progressCircle(file.fileStatus.sndProgress, file.fileStatus.sndTotal)
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/item/CIFileView.kt:159]: probably local files should have their own branch and always show empty view in place of status icon (they can't be transferred), and don't show "waiting" alerts programmed (they can't accepted) - here and same in CIImageView, CIVideoView

Comment (spaced4ndy) [reply apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/item/CIFileView.kt:159]: they're always created in sndStored status and if I remember correctly never change state

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/helpers/Utils.kt [hunk 7]
```diff
@@ -132,6 +132,8 @@ const val MAX_FILE_SIZE_SMP: Long = 8000000
 
 const val MAX_FILE_SIZE_XFTP: Long = 1_073_741_824 // 1GB
 
+const val MAX_FILE_SIZE_LOCAL: Long = 1_073_741_824 // 1GB
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/helpers/Utils.kt:135]: why is it Int64.max on ios but same as xftp here?

**Pull request merged**